csm_web/frontend/static/frontend/css/style.css

```diff
@@ -5,6 +5,7 @@
 @import url("./calendar.css");
 @import url("./searchbar.css");
 @import url("./tooltip.css");
+@import url("./policies-styles.css");
```

Comment:
The policies stylesheet should be named `policies.css` to comply more with the other stylesheet naming conventions.

Reply:
sg! file name changes